### PR TITLE
Add Team Constraints Documentation

### PR DESCRIPTION
**Pull request by Khusro-S**
---
name: Add Team Constraints Documentation 
about: This pull request merges the `Constraints` branch into `main`
---

## This PR includes the following changes:

- Added a detailed README/documentation outlining the constraints of the team.

## Impact
- No code or functional changes, this update is documentation only.

## Review Notes:

- Please check the README/documentation for accuracy, completeness, and clarity.
- Ensure it aligns with the team's agreed upon constraints.

### CODE REVIEW

File: collaboration/constraints.md [hunk 1]
```diff
@@ -1,34 +1,100 @@
...
+Welcome to the Constraints README! This document is here to help us understand
+the challenges and limitations each team member faces while working on this project.
+Since we're all learning and working remotely, there can be obstacles like different
+time zones 🌍, schedules ⏰, weak internet 🌐, power outages ⚡, or simply being
+new to GitHub 💻.
 
-## External
+By outlining these constraints, we aim to make collaboration easier and ensure
+everyone feels supported as we learn and grow together. Don’t worry, this is a
+beginner-friendly space, and we’re all here to help each other improve!
```
Comment (robiel0143) [inline collaboration/constraints.md:14]: Example 2  "add single comment" label for trial

Comment (Khusro-S) [reply collaboration/constraints.md:14]: testing

File: collaboration/constraints.md [hunk 2]
```diff
@@ -1,34 +1,100 @@
...
-  - technologies (sometimes a client will tell you what to use)
-  - power or connectivity
-  - ...
--->
+## Purpose 🎯
 
-## Internal: Involuntary
+The goal of this document is to identify and acknowledge the constraints we each
+face while working on this project. By being transparent about our challenges,
+we can plan better, support each other, and ensure smooth progress as a team.
```
Comment (robiel0143) [inline collaboration/constraints.md:20]: add single comment

**Review by robiel0143 — APPROVED** [review]

**Review by robiel0143 — APPROVED** [review]
Your code is approved.

**Review by Zeinab15 — APPROVED** [review]

**Review by robiel0143 — APPROVED** [review]
Well prepared

**Review by robiel0143 — APPROVED** [review]
Your project is well prepared!!

**Pull request merged**